builds/respec-worker.js

```diff
@@ -18,7 +18,7 @@ self.addEventListener("message", ({ data: originalData }) => {
       const { code } = data;
       const langs = data.languages.length ? data.languages : undefined;
       try {
-        const { value, language } = self.hljs.highlightAuto(code, langs);
+        const { value, language } = self.hljs.highlight(code, langs);
```

Comment:
I don't think this is right, the signature for highlight is: ` highlight(code, {language, ignoreIllegals})`.

Please see https://github.com/highlightjs/highlight.js/blob/main/VERSION_11_UPGRADE.md 

We need to decide if `ignoreIllegals` should be true or not.

Reply:
There should probably only be one language coming in too... you will need to check why we are sending in multiple languages. If we are sending in unknown languages, probably want to filter those out and we should only end up with 1 language. 

Additionally, you need to find a suitable default (something like "plaintext") - and we want "plaintext" to be included in the the highlighter build steps.